plugins/module_utils/kibana.py

```diff
@@ -12,6 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+from re import I
```

Comment:
`I` and `version` are imported but never used.

Reply:
Not sure why those are there then.  I didn't add them. Not on purpose anyway.

Reply:
If you are using vscode, I think it attempts to add missing modules. So if you typed 'I', it would search in modules for the 'I' module or function and add the import. That's been my experience anyways.